### DISCUSSION TITLE
Cannot access dashboard

### DISCUSSION THREAD

**Discussion by evanstoddard23** [renovatebot/renovate#8854] [Request Help]
**What Renovate type, platform and version are you using?**

hosted app

**Describe the bug**

Get GitHub and GitLab login options on dashboard. I choose GitHub and sign in, then am redirected back to the options page. Cannot get past this page on Chrome or Safari.

**Relevant debug logs**

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/master/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [ ] This is a really small bug, it does not need a reproduction (think small typo)
- [ ] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

**Comment by rarkins** [comment]
One of these normally works: Load developer tools and check the logs for something failing. Try another browser. Try with ad blocking removed.

**Reply by viceice** [reply]
![image](https://user-images.githubusercontent.com/1798109/109141330-e609d500-775d-11eb-8809-b8dd32bda41a.png)

**Reply by jdbruijn** [reply]
I'm getting similar, but on `user`.

![image](https://user-images.githubusercontent.com/62570005/109142950-c2478e80-775f-11eb-8f53-63ba3ecba418.png)

**Reply by viceice** [reply]
@jdbruijn it because you aren't loggind in, i had the login coockie already, so it fails on jobs for me 😉

**Comment by jdbruijn** [comment]
I'm getting the same since today, I was able to access it yesterday and know my Google Chrome didn't update as I'm on version `88.0.4324.150 (Official Build) (64-bit)`. Just tried this in Firefox and got the same issue.

**Reply by rarkins** [reply]
I can confirm/reproduce - taking a look

**Reply by rarkins** [reply]
Sorry, there was a problem with the backend server, which has been restarted. I can now log out / log in and retrieve jobs so I'm hoping it's fully fixed for you too.

**Reply by jdbruijn** [reply]
Can confirm! Thank you 🚀
